MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -615,6 +616,24 @@ public bool IsTextSubtitleStream
         /// <value><c>true</c> if this instance is anamorphic; otherwise, <c>false</c>.</value>
         public bool? IsAnamorphic { get; set; }
 
+        /// <summary>
+        /// Gets or sets compatibility errors.
+        /// </summary>
+        /// <value>Compatibility errors.</value>
+        public TranscodeReason? CompatibilityErrors { get; set; }
+
+        /// <summary>
+        /// Gets whether this instance is compatible with device.
+        /// </summary>
+        /// <value><c>true</c> if this instance is compatible with device; otherwise, <c>false</c>.</value>
+        public bool? IsCompatible
```

Comment:
Or maybe name them `DirectPlayErrors` and `SupportsDirectPlay` so that they correspond to `MediaSourceInfo.SupportsDirectPlay`? :thinking: